src/main/java/com/playmonumenta/networkchat/Message.java

```diff
@@ -144,6 +144,8 @@ private Message(UUID id,
 			}
```

Comment:
Note to self: this is the protected static method, not the private constructor. GitHub is lying.